### PR TITLE
Update dependencies to Clojure 1.7 and newest Compojure

### PR DESCRIPTION
**Pull request by bilus**

### CODE REVIEW

**Comment by bilus** [comment]
Maybe 1.7 should be optional but is there any reason still to default to 1.6?

**Comment by weavejester** [comment]
Nope, that's fine. Could you remove the ending period from your commit message, though?

**Comment by bilus** [comment]
Done.

File: resources/leiningen/new/compojure/project.clj [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [compojure "1.3.1"]
-                 [ring/ring-defaults "0.1.2"]]
-  :plugins [[lein-ring "0.8.13"]]
-  :ring {:handler {{namespace}}.handler/app}
+  :dependencies [[org.clojure/clojure "1.7.0"]
+                 [compojure "1.4.0"]
+                 [ring/ring-defaults "0.1.5"]]
+  :plugins [[lein-ring "0.9.7"]]
+  :ring {:handler {{namespace}.handler/app}}
```
Comment (weavejester) [inline resources/leiningen/new/compojure/project.clj]: Looks like you've accidentally moved the `}` here. You'll need to put it back, or the template won't work.

Comment (bilus) [inline resources/leiningen/new/compojure/project.clj]: Good catch! Sorry for that; paredit is not a silver bullet. :)

**Comment by weavejester** [comment]
Okay, that looks fine. Could you squash your commits together? Then I can merge.

**Comment by bilus** [comment]
Done. Sorry for the wait.

**Pull request merged**